### PR TITLE
Addind my art to the project

### PR DESCRIPTION
**Pull request by albertomtferreira**
Added my project to the list.

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -2853,6 +2853,13 @@ let cards = [
     imageLink: './Art/daniel-badura/emerald/emerald.gif',
     author: 'daniel-badura',
     githubLink: 'https://github.com/daniel-badura'
+  },
+  {
+    artName: 'Unknown', // change this to the name of your artwork
+    pageLink: './Art/albertomtferreira/index.html', // change this
+    imageLink: './Art/albertomtferreira/albertomtferreira.gif', // change this
+    author: 'Alberto Ferreira', // use your name
+    githubLink: 'https://github.com/albertomtferreira' // change this
```
Comment (sophiabrandt) [inline include.js:2862]: Can you please remove all the comments in this entry/

Comment (albertomtferreira) [reply include.js]: Done 👍

**Review by albertomtferreira — COMMENTED** [review]
Comments removed

**Review by albertomtferreira — COMMENTED** [review]
Not sure why I have a new reques!

**Comment by sophiabrandt** [comment]
@albertomtferreira Interesting. I assume it's because there were merge conflicts. Normally, it's on the person who makes the pull request to resolve those, but I have done that for you. The Zerobot is not very smart about this. You can ignore it.

**Pull request merged**